src/components/UIShared/LoadingPlaceholder.vue

```diff
@@ -156,6 +156,33 @@ export default {
 	}
 }
 
+// Event cards placeholder ruleset
+.placeholder-list--event-cards {
+	display: flex;
+	flex-wrap: nowrap;
+	.placeholder-item {
+		padding: 0;
+		margin: 0;
+		gap: 0;
+		&__content {
+			width: 100%;
+			padding: var(--default-grid-baseline);
+			margin: 0;
+			gap: var(--default-grid-baseline);
+
+			&-line {
+				margin: 0;
+				width: 100%;
+				min-width: 200px;
+				max-width: 300px;
```

Comment:
Since we have fancy buttons to scroll, maybe it's better to have a fixed width of 300px for cards? (and placeholders)?